### PR TITLE
294301

### PR DESCRIPTION
**Pull request by c00crane**
Split the com.ibm.ws.security.oidc.server_fat.jaxrs.config fat project.  Now that an EE10 repeat has been added, it takes too long to run and is timing out in FULL Mode.
I'll split the project into:
com.ibm.ws.security.oidc.server_fat.jaxrs.config.commonTest - tooling project that holds all of the test cases, configs, ...
com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth - a FATSuite.java that specifies the oAuth test classes to run
com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc - a FATSuite.java that specifies the OIDC test classes to run
com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP - a FATSuite.java that specifies the no OP test classes to run

### CODE REVIEW

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc,com.ibm.ws.security.oidc.server_fat.jaxrs.config
(expect com.ibm.ws.security.oidc.server_fat.jaxrs.config to fail since it shouldn't exist anymore)

**Comment by c00crane** [comment]
(expect com.ibm.ws.security.oidc.server_fat.jaxrs.config to fail since it shouldn't exist anymore)

#libby
#build
#soe.platform.requests=CENTOS8_X86_IBMJDK8_64_EBC
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc,com.ibm.ws.security.oidc.server_fat.jaxrs.config

**Review by ayoho — DISMISSED** [review]

**Comment by c00crane** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc

File: dev/com.ibm.ws.security.fat.common/src/com/ibm/ws/security/fat/common/actions/LargeProjectRepeatActions.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.fat.common/src/com/ibm/ws/security/fat/common/actions/LargeProjectRepeatActions.java]: The top section of the copyright needs to be updated to refer to v2.0: "Eclipse Public License 2.0" and "http://www.eclipse.org/legal/epl-2.0/"

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP/bnd.bnd]: These should refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP/build.gradle [hunk 3]
```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP/build.gradle]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP/fat/src/com/ibm/ws/security/openidconnect/server/fat/jaxrs/config/FATSuite.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP/fat/src/com/ibm/ws/security/openidconnect/server/fat/jaxrs/config/FATSuite.java]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth/bnd.bnd [hunk 5]
```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth/bnd.bnd]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth/build.gradle [hunk 6]
```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth/build.gradle]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth/fat/src/com/ibm/ws/security/openidconnect/server/fat/jaxrs/config/FATSuite.java [hunk 7]
```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth/fat/src/com/ibm/ws/security/openidconnect/server/fat/jaxrs/config/FATSuite.java]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc/bnd.bnd [hunk 8]
```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc/bnd.bnd]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc/build.gradle [hunk 9]
```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc/build.gradle]: Update to refer to v2.0.

File: dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc/fat/src/com/ibm/ws/security/openidconnect/server/fat/jaxrs/config/FATSuite.java [hunk 10]
```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc/fat/src/com/ibm/ws/security/openidconnect/server/fat/jaxrs/config/FATSuite.java]: Update to refer to v2.0.

**Comment by c00crane** [comment]
#libby

**Comment by c00crane** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc

**Review by ayoho — DISMISSED** [review]

**Comment by c00crane** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc

**Comment by c00crane** [comment]
Requesting yet another build since connections to libfsfe06 failed.

#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc

**Review by ayoho — DISMISSED** [review]

**Comment by c00crane** [comment]
Requesting yet another build because I lost part of the change when I refreshed the update to pick up the copyright update - The FATSuite should have been removed from the commonTest project.

#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.server_fat.jaxrs.config.noOP,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oauth,com.ibm.ws.security.oidc.server_fat.jaxrs.config.oidc

**Review by ayoho — DISMISSED** [review]

**Review by ayoho — APPROVED** [review]

**Comment by c00crane** [comment]
#libby

**Comment by c00crane** [comment]
#libby

**Pull request merged**